### ISSUE TITLE
Remove punctuation from rendered attributes

### ISSUE THREAD

**Issue by alvaromorales** [infolab-csail/WikipediaBase#128]
Attributes below should not end in colons.

```python
In [12]: get_meta_infobox('Template:Infobox road').attributes
Out[12]:
{u'allocation': u'Component highways:',
 u'cities': u'Major cities:',
 u'counties': u'Counties:',
 u'decommissioned': u'Existed:',
 u'deleted': u'Existed:',
 u'districts': u'Districts:',
 u'established': u'Existed:',
 u'formed': u'Existed:',
 u'history': u'History:',
 u'junction': u'',
 u'municipalities': u'Municipalities:',
 u'related': u'Related routes:',
 u'restrictions': u'Restrictions:',
 u'significance': u'Cultural significance:',
 u'terminus-a': u'From:',
 u'terminus-b': u'To:',
 u'time-period': u'Time period:',
 u'towns': u'Towns:',
 u'villages': u'Villages:'}
```

**Comment by fakedrake** [comment]
also

``` python
>>> get_meta_infobox('Template:Infobox settlement').attributes
{u'area-land-ha': u'\u2022 Land',
[...]
 u'government-type': u'\u2022 Type',
[...]
 u'area-urban-ha': u'\u2022 Urban',
 u'area-urban-acre': u'\u2022 Urban',
[...]
 u'length-km': u'\u2022 Length',
[...]
 u'area-rural-acre': u'\u2022 Rural',
 u'governing-body': u'\u2022 Body',
[...]
 u'population-total': u'\u2022 Total',
 u'grid-position': u'Grid position',
 u'area-blank2-ha': u'\u2022 {{{area_blank2_title}}}',
 u'area-metro-acre': u'\u2022 Metro',
 u'length-mi': u'\u2022 Length',
 u'population-metro': u'\u2022 Metro',
 u'population-density-rank': u'\u2022 Density rank',
 u'width-mi': u'\u2022 Width',
 u'population-est': u'\u2022 Estimate ({{{pop_est_as_of}}})',
 u'area-blank1-acre': u'\u2022 {{{area_blank1_title}}}',
 u'population-note': u'',
 u'area-code': u'Area code(s)',
 u'area-metro-sq-mi': u'\u2022 Metro',
 u'area-land-acre': u'\u2022 Land',
 u'parts': u'Boroughs',
 u'area-metro-ha': u'\u2022 Metro',
 u'area-rural-ha': u'\u2022 Rural',
 u'area-total-ha': u'\u2022 Total',
 u'population-rank': u'\u2022 Rank',
 u'geocode': u'Geocode',
 u'website': u'Website',
 u'population-rural': u'\u2022 Rural',
 u'area-land-sq-mi': u'\u2022 Land',
 u'population-demonym': u'Demonym(s)',
 u'width-km': u'\u2022 Width',
 u'area-water-acre': u'\u2022 Water',
 u'elevation-m': u'Elevation',
 u'area-blank1-sq-mi': u'\u2022 {{{area_blank1_title}}}',
 u'area-blank2-sq-mi': u'\u2022 {{{area_blank2_title}}}',
 u'area-blank1-ha': u'\u2022 {{{area_blank1_title}}}',
 u'area-total-sq-mi': u'\u2022 Total',
 u'area-rank': u'Area rank',
 u'area-urban-sq-mi': u'\u2022 Urban',
 u'area-note': u'',
 u'elevation-min-m': u'Lowest elevation',
 u'registration-plate': u'Vehicle registration',
 u'area-blank2-acre': u'\u2022 {{{area_blank2_title}}}',
 u'area-water-ha': u'\u2022 Water',
 u'area-total-acre': u'\u2022 Total',
 u'named-for': u'Named for',
 u'elevation-max-m': u'Highest elevation',
 u'population-urban': u'\u2022 Urban',
 u'elevation-max-ft': u'Highest elevation',
 u'area-rural-sq-mi': u'\u2022 Rural',
 u'area-water-sq-mi': u'\u2022 Water'}
```
See [Manchester](https://en.wikipedia.org/wiki/Manchester)

**Comment by fakedrake** [comment]
This may be part of a deeper issue. The `\2022` characters at the beginning of each attribute indicate a hierarchy that probably should not be ignored. For example we have

    >>> get_meta_infobox('Template:Infobox settlement').attributes
    {[...]
     u'area-total-acre': u'\u2022 Total',
     u'area-total-ha': u'\u2022 Total',
     u'area-total-sq-mi': u'\u2022 Total',
    [...]
     u'population-total': u'\u2022 Total',
    [...]
     }

It's ok to have `death-place` and `death-date` mapping to rendered `Died` because the map to the same table cell but `population-total` and `area-total-sq-mi` are very different things. I am not sure what implications this could have but it doesn't feel correct.

**Comment by alvaromorales** [comment]
Yes, this type of infobox with nested attributes is really hard to parse :cry: 

See #24 -- @cordeliaavery is working on a solution.
